docker-compose.yml

```diff
@@ -0,0 +1,37 @@
+version: '3.7'
+
+services:
+  api:
+    build: ./api
+    ports:
+      - 80:80
+      - 8000:8000
+      - 3306:3306
+    volumes:
+      - ./mysql:/var/lib/mysql
+
+  manager:
+    build:
+      context: ./client
+      dockerfile: Dockerfile_manager
+    ports:
+      - 3000:3000
+    working_dir: /app/service-manager
+    stdin_open: true
```

Comment:
How is this used?

Reply:
only included to address this bug in React -  https://github.com/facebook/create-react-app/issues/8688

added a comment